### DISCUSSION TITLE
Using GitHub Discussions

### DISCUSSION THREAD

**Discussion by salieflewis** [public-assembly/.github#5] [General]
Hi All.

I'd love to start using GitHub discussions more to bring the conversation closer to the source. I'm curious if anyone has any insights on this. One question I am toying with off the bat, is whether or not discussions should be opened directly in their related repositories, or if we keep all the energy here in the organization wide discussion. Just starting the conversation, give me your thoughts!

@projectmehari @gainor @mmsaki @thekidnamedkd

**Comment by calebguy** [comment]
Hey there! gooood thoughts -- I think to me, keeping discussions to repos specifically makes sense just purely based on information organization. From the perspective of someone who wants to contribute, but isn't as engaged in the forum or other comms, it feels easier to peep a repo for a project I'm interested in & checkout the discussions and/or issues to see if there are any tasks or convos I can pick up and contribute to!

**Reply by salieflewis** [reply]
I too think this is the way to move forward. And, very excited to get a response from you! I am curious about exactly that... do you have any insight as to what you might be interested in contributing towards?

**Reply by calebguy** [reply]
This is the worst of the worst answers I know, but anything and everything haha. Do you have anything in mind / any current projects where ya'll have engineering needs? Would love to help on MUTUAL if there is any movement there or adding proposal creation into flexible could be cool.

**Reply by losingmyego** [reply]
@gainor feel free to dm me about contributing to MUTUAL. Very much looking for some peeps to help out. @losingmyego everywhere

**Reply by salieflewis** [reply]
There are some existing tickets on Flexible if you're interested in getting your hands dirty over there. Some of the performance stuff is more related to dao-utils which I'm in the process of composing another proposal surrounding. 

@0xTranqui introduced the idea of potentially seeking funding via Entropy for continued work on Flexible, which is exactly the nature I'd hope the development took. Once it becomes shared infrastructure amongst multiple DAOs, we can co-share responsibility in developing it out further.

I too am happy to chat about whatever and help you get up to speed with whatever you end up being interested in!

**Reply by calebguy** [reply]
Awesome, that sounds great. I'll peep the tickets over there & will ping ya if I have any questions. 

RE: future funding for Flexible -- I think thats a great idea. I've been mulling on a future proposal for Entropy for some product related things and I think adding some Flexible dev in the mix makes a lot of sense!

Thanks!

**Reply by salieflewis** [reply]
Great! The beginning of some conversations about potential directions for future development are [here](https://forum.public---assembly.com/t/first-fork-of-flexible/392/1) in the forum.

**Comment by projectmehari** [comment]
Hey everyone;

On my end to echo Caleb thoughts, I think the forum can serve as
transparent entry point for anyone who wants take a look at the macro
activities of PA.

For specific discussions on GitHub tasks/repo work, it easier for for me
chime in direct here. It feels more straight to the point. As contributor,
I can add my input and have direct focus on the tasks at hand.

Forum = macro Public Assembly organization

GitHub =   Project specific discussions

On Tue, Apr 25, 2023 at 10:14 Salief Lewis ***@***.***> wrote:

> Hi All.
>
> I'd love to start using GitHub discussions more to bring the conversation
> closer to the source. I'm curious if anyone has any insights on this. One
> question I am toying with off the bat, is whether or not discussions should
> be opened directly in their related repositories, or if we keep all the
> energy here in the organization wide discussion. Just starting the
> conversation, give me your thoughts!
>
> @projectmehari <https://github.com/projectmehari> @gainor
> <https://github.com/gainor> @mmsaki <https://github.com/mmsaki>
> @thekidnamedkd <https://github.com/thekidnamedkd>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/public-assembly/discussions/5>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR4FJL4U7IIULJ765BIFSZ3XC7L4PANCNFSM6AAAAAAXLCJ35Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 


Kirubel Mehari
Artist/Creative technologist
Phone:  514-795-5216
Site:  Projectmehari.com <http://projectmehari.com/>
[image: created with MySignature.io] <http://verymehari/>
<https://mysignature.io/editor/?utm_source=freepixel>
